README.md

```diff
@@ -49,96 +30,98 @@ Via Bower:
 
 The adal.js source is [here](https://github.com/AzureAD/azure-activedirectory-library-for-js/tree/master/lib/adal.js).
 The adal-angular.js source is [here](https://github.com/AzureAD/azure-activedirectory-library-for-js/tree/master/lib/adal-angular.js).
-## Samples, tests and documentation
 
-For a sample demonstrating basic usage of ADAL JS please refer to [this repo](https://github.com/AzureADSamples/SinglePageApp-DotNet).
+## Build and run tests
 
-CORS API [sample ](https://github.com/AzureADSamples/SinglePageApp-WebAPI-AngularJS-DotNet).
-
-**To run tests**
+**Run tests**
 
     npm install
     bower install
     npm test
+
     // angular tests
     karma start
 
-Karma as test runner:
-You need to install the karma command line.
+To use Karma as test runner, you need to install the karma command line.
 
     npm install -g karma
     npm install -g karma-cli
 
-**Logging**
+**Reference doc generation**
 
-Log levels are mapped as:
+Install grunt and run the command
 
-    0: Error
-    1: Warning
-    2: Info
-    3: Verbose
+    grunt doc
 
-You can add the code below to app.js to turn on logging. Implement the `log` method depending on how you want to redirect logs.
+## Usage
 
-    Logging = {
-        level: 3,
-        log: function (message) {
-            console.log(message);
-        }
-    };
-    
-**documentation generation**
-Install grunt; call
+**In JavaScript**
 
-    grunt doc
+You can use ADAL JS as follows in a plain JavaScript application without any frameworks.
+
+1- Include a reference to adal.js in your main app page before your application scripts.
+
+```html
+<script src="App/Scripts/adal.js"></script>
+<script src="App/Scripts/app.js"></script>
+```
+
+2- Initialize ADAL with the AAD app coordinates at app config time. The minimum required config to initialize ADAL is:
+```js
+window.config = {
+   clientId: 'g075edef-0efa-453b-997b-de1337c29185'
+};
+var authContext = new AuthenticationContext(config);
+```
+
+3- You can trigger the login and logout using the authContext
+```js
+$signInButton.click(function () {
+    authContext.login();
+});
 
+$signOutButton.click(function () {
+    authContext.logOut();
+});
+```
```

Comment:
Would it make sense to add a link to have a minimal devApps in the repo and point to it as a super simple sample (like what you have for msal.js):
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/devApps/VanillaJSTestApp/index.html
?

Reply:
Currently there is no dev sample in ADAL.js repo. But I can link to the plain JS singlepageapp sample in our samples directory.